src/main/java/be/winagent/weba2/controllers/HomeController.java

```diff
@@ -39,4 +40,11 @@ public String home(@ModelAttribute(name = "currentUser") User currentUser, Model
         model.addAttribute("adminAssociations", adminAssociations);
         return "home";
     }
+
+    @GetMapping("/info")
+    @Authenticated
+    public String info(@RequestParam(required = false) String lang, @ModelAttribute(name = "currentUser") User currentUser) {
```

Comment:
What is this `lang` parameter for? Won't this conflict with the parameter for the switching locale?

If you need the locale for the current user, just add `Locale locale` as a method param.
https://stackoverflow.com/a/33050046